koji/README.md

```diff
@@ -0,0 +1,20 @@
+---
+title: koji
+homepage: https://github.com/its-danny/koji
+tagline: |
+  🦊 An interactive CLI for creating conventional commits. 
+---
+
+To update or switch versions, run `webi koji@stable` (or `@v2`, `@beta`, etc).
+
+## How to run `koji`
+
+```bash
+koji
+
+# with emoji
+koji --emoji
```

Comment:
I want a little more detail. I can't tell from this alone how to use this command. Is it a substitute for `git commit`?

Or do I run `git commit -m "do some stuff whatever"` and then this adds the prefix?